### PR TITLE
Enforce newline-per-chained-call rule and fix code style

### LINKED ISSUE #1019: [DevEx]: Configure eslint rule: newline-per-chained-call
### Improvement Summary

In order to reduce stylistic discrepancies between contributors regarding line length, we should apply the newline-per-chained-call eslint rule

### Detailed Description

Here is the rule definition: https://eslint.style/rules/js/newline-per-chained-call. We should use the default value of 2 for `ignoreChainWithDepth` to allow short chains of 2 commands to use a single line. We should run `npm run lint:fix` and commit the changes after configuring the rule to normali3 line length across the project.

This may require installing an additional dev dependency for style rules.

### PR DESCRIPTION
**Pull request by Mang0codes**
## Issue number

Relevant [issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #1019

## Please check the following

- [ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

### CODE REVIEW

**Review by itsalaidbacklife — CHANGES_REQUESTED** [review]
This looks like the right idea but something is going wrong for it to say so many things are changing. It looks like the affected files are completely rewritten instead of having individual lines changed

File: .eslintrc.js
[the remark this answers is not in the document]
Comment (Mang0codes) [reply .eslintrc.js:1]: I think the issue might be with the Windows machine. You can check the latest commit to see the changes, or I can raise another PR if you'd prefer.

[the remark this answers is not in the document]
Comment (itsalaidbacklife) [reply .eslintrc.js:1]: @Mang0codes that makes sense. It is something that I thought/hoped we'd standardized such that this wouldn't happen when windows users submit updates, but it looks like either it wasn't previously done, or wasn't done correctly. For the moment, could you try reimplementing the change using WSL (windows subsystem linux) to standardize line endings to the unix ones)? 

I'm writing up an issue to fix this

[the remark this answers is not in the document]
Comment (itsalaidbacklife) [reply .eslintrc.js:1]: @Mang0codes I've written up #1082 to fix this issue. If you'd like, you could tackle that issue first yourself and then that might fix this issue here

**Pull request closed without merging**